lib/diagnostic.rb

```diff
@@ -13,8 +13,12 @@
 # and location should only be writable.
 
 ##
-# your code here
-##
+class Person
+  initialize(name, age, location)
+  attr_reader :age ##
+  attr_writer :location
+  attr_accessor :name
+end
 
```

Comment:
The `attr_reader/writer/accessor` should go above the initialize method. Then you have to initialize instance variables within the method

```ruby
def initialize (name, age, location)
    @name, @age, @location = name, age, location
end
```